.github/workflows/test_suite_ubuntu.yml

```diff
@@ -45,48 +45,26 @@ jobs:
       - name: Install Python
         uses: actions/setup-python@v5
         with:
-          python-version: '>=3.7,<3.11'
-          check-latest: false
-          token: ***
-          update-environment: true
-          allow-prereleases: false
+          python-version: '3.x'
 
       # Install dependencies
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y libhdf5-dev libnetcdf-dev
+          sudo apt-get install -y libhdf5-dev libnetcdf-dev libnetcdff-dev
         
-      # Generate sounding files
-      - name: Generate sounding files 
-        run: |
-          cd YOG_convection/resources/
-          python3 -m venv venv
-          source venv/bin/activate
-          python3 -m pip install -r requirements.txt
-          python3 sounding_to_netcdf.py
-          deactivate
-          cd ../../
-
-      # Add Python linter which is ruff
-      - name: ruff
-        if: always()
-        run:  |
-          cd ${{ github.workspace }}
-          ruff format --diff $(git ls-files *.py)
-          ruff check --diff $(git ls-files *.py)
-
       # Build the code with cmake
       - name: Build test suite
         run:  |
-          export TEST_DIR=$(pwd)/tests
-          cd ${TEST_DIR}
+          cd tests/
           mkdir build
           cd build
           cmake ..
           cmake --build .
+          ./test_CAM_interface
```

Comment:
```suggestion
```

Reply:
This is the build part, it's run below

Reply:
Thanks for that catch, it arose because I was playing around with trying to work out if the runner reset the location to `~/` at the start of each step, and my conclusion is that it does (I think??). Will commit.